src/pykiso/logging_initializer.py

```diff
@@ -234,3 +239,94 @@ def initialize_loggers(loggers: Optional[List[str]]) -> None:
     loggers_to_deactivate = set(relevant_loggers) - set(active_loggers)
     for logger_name in loggers_to_deactivate:
         logging.getLogger(logger_name).setLevel(logging.WARNING)
+
+
+def import_object(path: str) -> Any:
+    """return the object based on the path.
+        For example : logging.Logger will return Logger
+
+    :param path: path to the object
+    """
+    if path:
+        components = path.split(".")
+        mod = importlib.import_module(components[0])
```

Comment:
You could also make use of the package kwarg:
```
components = path.split(".")
if len(components) == 1:
    return importlib.import_module(components[0])
else:
    return importlib.import_module(components[-1], package=component[:-1])

Reply:
What I want in the end is not a module but an Object.

Reply:
Then a single getattr should be sufficient

Reply:
Done